packages/date-picker/test/validation.common.js

```diff
@@ -209,14 +151,6 @@ describe('validation', () => {
       expect(datePicker.invalid).to.be.false;
     });
 
-    it('should keep invalid input value when closing overlay', async () => {
```

Comment:
**note:** Tested within the [new "bad input" suite](https://github.com/vaadin/web-components/pull/6313/files#diff-2e12dbf30265bd94f0b4aec2f7e30c9fc315e508b10aee21647aa3a09670e8f2R100-R136).